Assets/Scripts/World/Units/Enemy/Main/EnemyController.cs

```diff
@@ -72,6 +74,7 @@ public class EnemyController : UnitController<IEnemy, Enemy, EnemyList>, IEnemyC
 		if (orderedEnemyPrefabs.ContainsKey(enemyKey)) {
 			GameObject enemy = (GameObject) Instantiate(orderedEnemyPrefabs[enemyKey], EnemySpawnPoint.GetPosition(spawnDirection), Quaternion.identity);
 			EnemyBehaviour enemyBehaviour = enemy.GetComponent<EnemyBehaviour>();
+            AddActiveEnemy(enemyBehaviour);
```

Comment:
EnemyBehaviour. You can put it inside the "CleanupReferences" method (which is called when the enemy is being destroyed").